requirements/extras.txt

```diff
@@ -2,4 +2,5 @@ comet-ml>=3.31.7
 gradio>=2.9.4
 tensorboard
 wandb==0.12.17
-ipykernel
\ No newline at end of file
+GitPython
```

Comment:
What is this used for?

Reply:
There is this line `from git.repo import Repo` which requires gitpython